packit_service/worker/helpers/fedora_ci.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+
+from ogr.abstract import GitProject
+
+from packit_service.worker.events import EventData
+from packit_service.worker.reporting import BaseCommitStatus
+from packit_service.worker.reporting.reporters.base import StatusReporter
+
+logger = logging.getLogger(__name__)
+
+
+class FedoraCIHelper:
+    status_name: str = "Packit - scratch build"
```

Comment:
I am not sure I understand the need for a new helper. 

The ` KojiBuildJobHelper` has some more methods that could be useful when building koji builds that are missing in the generic `FedoraCIHelper` (if I am not wrong).
If we need the FedoraCIHelper because of the reporting (that is different in Pagure) then I am wondering if we could factor it out in the `StatusReporter` or somewhere else related with the reporting. And when creating the helpers (that we already have) we can choose which kind of reporter inject?

I just fear that we will need to duplicate all our helpers, creating a new "base".

Reply:
yes, at the moment, it really mostly wraps up the status reporting, in my initial implementation it contained some other logic common for the 2 new handlers. So that was the main idea, to have the common logic in here. Other thing is that I didn't want to adjust too much of existing code since this is more of a prototype and might be later changed/moved, therefore I wanted to keep it in one separate place. But having it as it is now, I am ok to just repeat the status reporting logic for the 2 handlers.

Reply:
It does not need to be done now, but I agree with Maja that we should be cautious about adding more helpers. (TBH, I don't think this was a good idea when introducing them... yes, I am probably the one to blame..;)